raiden/utils/typing.py

```diff
@@ -1,10 +1,30 @@
 # -*- coding: utf-8 -*-
-from typing import *  # NOQA
+from typing import *  # NOQA pylint:disable=wildcard-import,unused-wildcard-import
+from typing import NewType
 
-Address = bytes
-Secret = bytes
-Signature = bytes
-Keccak256 = bytes
+T_Address = bytes
+Address = NewType('Address', bytes)
 
-BlockNumber = int
-TokenAmount = int
+T_BlockExpiration = int
+BlockExpiration = NewType('BlockExpiration', int)
+
+T_BlockNumber = int
+BlockNumber = NewType('BlockNumber', int)
+
+T_BlockTimeout = int
+BlockTimeout = NewType('BlockNumber', int)
+
+T_ChannelID = T_Address
+ChannelID = NewType('ChannelID', Address)
```

Comment:
This is wrong.

`NewType` returns a function, so `Address` is a function.

> However, at runtime, NewType('Derived', Base) will return a dummy function that simply returns its argument

And the docs says:

> The second argument must be a properly subclassable class, i.e., not a type construct like Union, etc.

which `function` is not.

https://mypy.readthedocs.io/en/latest/kinds_of_types.html#newtypes